packages/controller-config/src/docker-images.json

```diff
@@ -17,7 +17,7 @@
   "kubeRBACProxy": "quay.io/brancz/kube-rbac-proxy:v0.8.0",
   "kubeStateMetrics": "quay.io/coreos/kube-state-metrics:v1.7.2",
   "localVolumeProvisioner": "quay.io/external_storage/local-volume-provisioner:v2.2.0",
-  "loki": "grafana/loki:2.2.0",
+  "loki": "grafana/loki:2.2.1",
```

Comment:
So, this release is from April 6. While we are here: what do we think about leaping to latest main, to include April/May/June development? 

https://github.com/grafana/loki/compare/v2.2.1...main

Reply:
Let's do it separately because we had 2.2.1 run in CI for almost two weeks before the downgrade without any new symptoms.